app/models/source.rb

```diff
@@ -11,6 +11,7 @@ class Source < ApplicationRecord
 
   attribute :availability_status, :string
   validates :availability_status, :inclusion => { :in => %w[available partially_available unavailable] }, :allow_nil => true
+  validates :app_creation_workflow, :inclusion => {:in => %w[paranoid semi-trust trust]}
```

Comment:
I would change these values to something more neutral, as it is can be still read by users in their network tabs.

`manual semi-auto auto`

or we can call it the same as in the UI

`manual-configuration account-authorization` 

Also, there is no semi-trust mode, maybe we could skip for now.

Reply:
I would like to have these values equal with UI to avoid double-naming confusion, sync with UX could be helpful

Reply:
Will update it to match the UI, we can add semi-trust when necessary.